### PR TITLE
Build release

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @joshfree, @vsadov please review.
@MattGertz, @ManishJayaswal for approval.

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
Looks like CI is green.  @cston do you need to do anything to get the packages stable as well?  Looks like this is just changing dependencies.

**Comment by tmat** [comment]
:+1: 
Have you tried to build Roslyn nugets, add references to a dotnet cli app and restore?

How to do that:

- Build Roslyn.sln
- ```cd C:\Roslyn\build```
- ```csi BuildNuGets.csx C:\Roslyn\Binaries\Debug 9.9.9.9 C:\Roslyn\Binaries\Debug\Nuget```

**Comment by ericstj** [comment]
@tmat are you asking me to do that?  I'm not about to manually build Roslyn's shipping nuget packages on my dev box.  We need stable 1.3 packages out of your build workflow.

**Comment by tmat** [comment]
I was asking @cston

File: build/Targets/VSL.Versions.targets [hunk 1]
```diff
@@ -7,15 +7,15 @@
     <SystemReflectionMetadataVersion>1.2.0</SystemReflectionMetadataVersion>
     <SystemCollectionsImmutableVersion>1.1.37</SystemCollectionsImmutableVersion>
     <MicrosoftDiaSymReaderVersion>1.0.7</MicrosoftDiaSymReaderVersion>
-    <MicrosoftDiaSymReaderNativeVersion>1.4.0-rc2</MicrosoftDiaSymReaderNativeVersion>
+    <MicrosoftDiaSymReaderNativeVersion>1.4.0-rc2</MicrosoftDiaSymReaderNativeVersion> <!-- TODO -->
     <MicrosoftDiaSymReaderPortablePdbVersion>1.0.0</MicrosoftDiaSymReaderPortablePdbVersion>
-    <MicrosoftCodeAnalysisElfieVersion>0.10.6-rc2</MicrosoftCodeAnalysisElfieVersion>
+    <MicrosoftCodeAnalysisElfieVersion>0.10.6</MicrosoftCodeAnalysisElfieVersion>
```
Comment (ericstj) [inline build/Targets/VSL.Versions.targets]: These two packages may not be needed to go stable. (DiaSymReader and Elfie).

**Comment by cston** [comment]
@tmat, @ericstj Updated the PR based on feedback.

**Pull request merged**